test/e2e/config/azure-dev.yaml

```
@@ -206,14 +206,14 @@ providers:
 variables:
   AKS_KUBERNETES_VERSION: "latest"
   AKS_KUBERNETES_VERSION_UPGRADE_FROM: "latest-1"
-  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.29}"
+  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.32}"
   FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.26}"
-  KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.28.10"
+  KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.31.10"
```

This is the latest patch image we've built for 1.31:
```suggestion
  KUBERNETES_VERSION_API_UPGRADE_FROM: "v1.31.9"
```